Guided Investigation - WAF data.ipynb

```diff
@@ -0,0 +1,4156 @@
+{
```

Comment:
<p>Some of these long lines are tricky to read. If you're using Jupyter lab (which you should since it's a 1000 times better than notebook), there is a great code formatting extension https://github.com/ryantam626/jupyterlab_code_formatter - you need to pip install black as well. It can reformat the entire notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>